__tests__/integration.test.js

```diff
@@ -123,4 +123,62 @@ describe("GET requests", () => {
         });
     });
   });
+
+  describe("/api/articles", () => {
+    it('returns with 200 status code "OK"', () => {
+      return request(app).get("/api/articles").expect(200);
+    });
+
+    it("responds with an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(Array.isArray(articles)).toBe(true);
+          expect(articles.length).toBeGreaterThan(0);
+
+          for (let i = 0; i < articles.length; i++) {
+            const isObject = Object.keys(articles[i]).length > 0;
+            const isNotArray = !Array.isArray(articles[i]);
+
+            expect(isObject && isNotArray).toBe(true);
+          }
+        });
+    });
+
+    it("returned objects have a call_count property", () => {
+      return request(app)
+        .get("/api/articles")
+        .then(({ body: { articles } }) => {
+          for (let i = 0; i < articles.length; i++) {
+            expect(articles[i].hasOwnProperty("comment_count")).toBe(true);
+          }
+        });
```

Comment:
We can tidy this I think. 

Try using a forEach then checking the articles using...
expect.objectContaining({
    article_id: expect.any(Number),
    etc...